### PR TITLE
PS - added loader from sequence

### PR DESCRIPTION
**Pull request by kalisp**
Closes https://github.com/pypeclub/client/issues/46

Requires: https://github.com/pypeclub/OpenPype/pull/1726

### CODE REVIEW

File: avalon/pipeline.py [hunk 1]
```diff
@@ -192,10 +192,14 @@ class Loader(list):
...
 
+    options = []
+
     def __init__(self, context):
-        self.fname = self.filepath_from_context(context)
+        self.fname = Loader.filepath_from_context(context)
 
-    def filepath_from_context(self, context):
+    @classmethod
+    def filepath_from_context(cls, context):
```
Comment (iLLiCiTiT) [inline avalon/pipeline.py:201]: Why this became classmethod?

File: avalon/tools/loader/widgets.py [hunk 2]
```diff
@@ -1257,14 +1282,24 @@ def change_visibility(self, column_name, visible):
...
+                (containing repre_doc, version_doc, subset_doc, project info)
+            options (dict) - qargparse arguments to fill OptionDialog
+            data_by_repre_id (dict) - additional data applicable on top of
+                options to provide dynamic values
+    """
     error_info = []
-    repre_contexts = pipeline.get_repres_contexts(repre_ids, dbcon)
 
-    options = options or {}
+    if options is None:  # not load when cancelled
```
Comment (iLLiCiTiT) [inline avalon/tools/loader/widgets.py:1300]: In that case `options` is required argument of function and should not have default value `None`.

File: avalon/pipeline.py [hunk 3]
```diff
@@ -246,6 +250,17 @@ def remove(self, container):
...
+    def get_options(cls, contexts):
+        """
+            Returns static (cls) options or could collect from 'contexts'.
+
+            Args:
+                contexts (list): of repre or subset contexts
+            Returns:
+                (list)
+        """
+        return cls.options
```
Comment (iLLiCiTiT) [inline avalon/pipeline.py:263]: Maybe should check if `cls.options is None`. For cases when somebody will set the attribute value to `None` (to turn of options?) as `None` is used as checked for existence of options and would invalidate whole loading and skip the processing without telling what happened.

File: avalon/pipeline.py (hunk 1 above)
Comment (kalisp) [reply avalon/pipeline.py]: My bad, `staticmethod` would be better. The method is not changing state and could be useful event without initialized object.

Comment (iLLiCiTiT) [reply avalon/pipeline.py]: But there are some Loaders that have overriden this method and static definition in default will break them... Keep it as object mehod.

**Review by 64qam — APPROVED** [review]

**Pull request merged**